### DISCUSSION TITLE
PIP-18: Clock Offset Calculation

### DISCUSSION THREAD

**Discussion by ja7ad** [pactus-project/PIPs#79] [PIPs]
PIP-18: [Clock Offset Calculation](https://pips.pactus.org/PIPs/pip-18)

**Comment by b00f** [comment]
Some useful links:

- https://www.ntppool.org/en/use.html
- https://github.com/beevik/ntp
- https://en.wikipedia.org/wiki/Network_Time_Protocol
- https://datatracker.ietf.org/doc/html/rfc5905

**Reply by ja7ad** [reply]
@b00f This proposal it's okay for implementation, are you accept it?
